src/index.html

```diff
@@ -6,14 +6,483 @@
+                    ></div>
+                    <div
+                      class="testimonial-card__square testimonial-card__square--right"
+                    ></div>
+                  </div>
+                  <img
+                    src="./images/testimonials/testimonials-2.png"
+                    alt="photo"
+                    class="testimonial-card__photo"
+                  />
+                </div>
+                <div class="testimonial-card__quot"></div>
+                <p class="testimonial-card__text">AIR is an exceptional agency that leads with creative talent, first-class account servicing.</p>
+                <h3 class="testimonial-card__name">Azadeh Hawkins</h3>
+                <h4 class="testimonial-card__profession">Hawkins Consulting</h4>
+              </article>
+
+              <article class="testimonial-card testimonials__single-wrapped">
+                <div class="testimonial-card__image">
+                  <div class="testimonial-card__squares">
+                    <div
+                      class="testimonial-card__square testimonial-card__square--left"
+                    ></div>
+                    <div
+                      class="testimonial-card__square testimonial-card__square--right"
+                    ></div>
+                  </div>
+                  <img
+                    src="./images/testimonials/testimonials-3.png"
+                    alt="photo"
+                    class="testimonial-card__photo"
+                  />
+                </div>
+                <div class="testimonial-card__quot"></div>
+                <p class="testimonial-card__text">AIR raises the agency bar to stratospheric heights on both creative output and client service.</p>
+                <h3 class="testimonial-card__name">Michel Grover</h3>
+                <h4 class="testimonial-card__profession">Hulu</h4>
+              </article>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="result">
+        <div class="container">
+          <h2 class="result__title">Vision, Passion, Results</h2>
+          <p class="result__text">We are sure that first-rate job is possible only if all three components are united.</p>
+          <div class="result__button">
+            <a
+              href="#contact-form"
+              class="button"
+            >
+              Apply
+            </a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <form
+            class="contact-form"
+            id="contact-form"
+          >
+            <label
+              class="contact-form__label"
+              for="contact-form"
+            >
+              Send us a message
+            </label>
+            <input
+              type="email"
+              name="email"
+              placeholder="Your email"
```

Comment:
Placeholder should be an example of a valid value for this input to allow the user to easily visualize the expected format [Reference](https://github.com/HigorPedroso/layout_dia/tree/develop#:~:text=Placeholders%20in%20the%20forms%20suggest%20what%20to%20enter%2C%20and%20if%20there%20is%20a%20validation%20of%20the%20form%2C%20then%20it%20is%20clear%20in%20what%20format%20to%20enter%20the%20phone%20number)